cmd/edenPacket.go

```diff
@@ -53,7 +53,7 @@ func newPacketRunCmd(cfg *openevec.EdenSetupArgs, packetKey, packetProjectName *
 		Use:   "run",
 		Short: "run vm in packet",
 		Run: func(cmd *cobra.Command, args []string) {
-			err := openevec.PacketRun(*packetKey, *packetProjectName, packetVMName, packetZone, packetMachineType, packetIPXEUrl, cfg)
```

Comment:
Seems like you don't need cfg in function anymore

Reply:
Sorry, can you please elaborate, what exactly you want to see here?

Reply:
Unfortunately, I can't commit to your PR :D What I meant is yetus is complaining on unused variable, we can fix it by

```
diff --git a/cmd/edenPacket.go b/cmd/edenPacket.go
index 3f0e22e..82c60c1 100644
--- a/cmd/edenPacket.go
+++ b/cmd/edenPacket.go
@@ -16,7 +16,7 @@ func newPacketCmd(configName, verbosity *string) *cobra.Command {
                Short: `Manage VMs in Equinix Metal Platform`,
        }
 
-       packetCmd.AddCommand(newPacketVMCmd(cfg, &cfg.Packet.Key, &packetProjectName))
+       packetCmd.AddCommand(newPacketVMCmd(&cfg.Packet.Key, &packetProjectName))
 
        packetCmd.PersistentFlags().StringVarP(&cfg.Packet.Key, "key", "k", "", "packet key file")
        packetCmd.PersistentFlags().StringVarP(&packetProjectName, "project", "p", defaults.DefaultPacketProjectName, "project name on packet")
@@ -24,7 +24,7 @@ func newPacketCmd(configName, verbosity *string) *cobra.Command {
        return packetCmd
 }
 
-func newPacketVMCmd(cfg *openevec.EdenSetupArgs, packetKey, packetProjectName *string) *cobra.Command {
+func newPacketVMCmd(packetKey, packetProjectName *string) *cobra.Command {
        var packetVMCmd = &cobra.Command{
                Use:   "vm",
                Short: `Manage VMs in packet`,
@@ -34,7 +34,7 @@ func newPacketVMCmd(cfg *openevec.EdenSetupArgs, packetKey, packetProjectName *s
                {
                        Message: "Basic Commands",
                        Commands: []*cobra.Command{
-                               newPacketRunCmd(cfg, packetKey, packetProjectName),
+                               newPacketRunCmd(packetKey, packetProjectName),
                                newPacketDeleteCmd(packetKey, packetProjectName),
                                newPacketGetIPCmd(packetKey, packetProjectName),
                        },
@@ -46,7 +46,7 @@ func newPacketVMCmd(cfg *openevec.EdenSetupArgs, packetKey, packetProjectName *s
        return packetVMCmd
 }
```